packages/theme/css/characteristics/text.css

```diff
@@ -61,18 +61,18 @@
   /* Colors */
   --salt-text-background-selected: var(--salt-palette-interact-background-active);
 
-  --salt-text-primary-foreground: var(--salt-palette-neutral-primary-foreground);
```

Comment:
I think these were intended to be in 'foreground' characteristic theme/css/characteristics/foreground

Reply:
ah yes great spot missed these out, moved them into their own files now